sirepo/package_data/static/js/activait.js

```diff
@@ -249,6 +259,14 @@ SIREPO.app.directive('appHeader', function(appState, activaitService) {
                 }
                 return false;
             };
+            $scope.isImageToImage = () => {
+                var info = appState.models.columnInfo;
+                if (info) {
+                    var idx = info.inputOutput.indexOf('output');
+                    return info.shape[idx].slice(1, info.shape[idx].length).length > 1;
+                }
+                return false;
```

Comment:
In the pets example, selecting the checkboxes in the Columns panel is showing a "undefined reading 'slice'" javascript error